core/controllers/topics_and_skills_dashboard.py

```diff
@@ -141,6 +143,7 @@ def post(self):
         linked_topic_ids = self.payload.get('linked_topic_ids')
         explanation_dict = self.payload.get('explanation_dict')
         rubrics = self.payload.get('rubrics')
+        filenames = self.payload.get('filenames')
```

Comment:
(I'm not sure) Do we still need this filename as a payload? COnsidering we can fetch all the filenames through the skill object and the `self.request.get(filename)` can provide images for each of those filenames, right? [Any specific reason for having this extra field?]

Reply:
Yeah, it's no longer needed, removed.